tests/platform_tests/api/test_sfp.py

```diff
@@ -145,6 +145,13 @@ class TestSfpApi(PlatformApiTestBase):
     # Helper functions
     #
 
+    def is_xcvr_dac(self, xcvr_info_dict):
```

Comment:
please reuse or improve [is_xcvr_optical()](https://github.com/Azure/sonic-mgmt/blob/02d630740fd9f752aa2427e059876622f199e127/tests/platform_tests/api/test_sfp.py#L148) instead of introducing one more API.